ecs-init/docker/docker.go

```diff
@@ -108,15 +108,17 @@ const (
 	iptablesLib64Dir    = "/lib64"
 	iptablesUsrLib64Dir = "/usr/lib64"
 
-	hostCapabilitiesResourcesRootDir      = "/var/lib/ecs"
+	hostCapabilitiesResourcesRootDir      = "/var/lib/ecs/deps"
 	containerCapabilitiesResourcesRootDir = "/capabilities"
 
-	capabilityExecName                       = "exec"
-	capabilityExecHostBinRelativePath        = "ssm-agent/linux_amd64"
-	capabilityExecContainerBinRelativePath   = "bin"
-	capabilityExecHostCertsDir               = "/etc/pki/ca-trust/extracted/pem"
-	capabilityExecContainerCertsRelativePath = "certs"
-	capabilityExecRequiredCert               = "tls-ca-bundle.pem"
+	capabilityExecName                        = "execute-command"
+	capabilityExecHostBinRelativePath         = "bin"
+	capabilityExecContainerBinRelativePath    = "bin"
```

Comment:
hmm, I don't think we will ever want to diverge from this... not opposed to ship this tho, but don't think it's necessary to have two `"bin"` and two `"config"`

Reply:
was not sure about this either, thanks for pointing it out